ignite/pkg/cosmosanalysis/app/app_test.go

```diff
@@ -128,6 +91,26 @@ func TestParseAppModules(t *testing.T) {
 			path:            "testdata/modules/arguments",
 			expectedModules: basicModules,
 		},
+		{
+			name:            "cosmos-sdk/types/module with alias",
+			path:            "testdata/modules/package_alias",
+			expectedModules: basicModules,
+		},
+		{
+			name:            "package not called app",
+			path:            "testdata/modules/package_not_called_app",
+			expectedModules: basicModules,
+		},
```

Comment:
What is this case testing?

Reply:
Since `FindRegisteredModules` now parses the whole `app` package and not only `app/app.go`, I have to select a package from a map.

 `parser.ParseDir()` returns a map where the key is the package name and the value is the `*ast.Package`, it's possible to have multiple packages in the same directory when you have `app` and `app_test` for instance. So initially I went for a good old `pkgs["app"]` to get the `app` package but this fails for `gaia`, because in the `gaia` repo, the package in `app` folder is named `gaia` !

So I had to write a loop on the map where I return the first package that doesn't have the `*_test` suffix. I decided to put everything in `xast.ParseDir` because I think most of the time you don't want that `_test` package.